client/my-sites/domains/controller.jsx

```diff
@@ -309,7 +279,9 @@ const redirectToUseYourDomainIfVipSite = () => {
 
 		if ( selectedSite && selectedSite.is_vip ) {
 			return page.redirect(
-				domainUseYourDomain( selectedSite.slug, get( context, 'params.suggestion', '' ) )
+				domainUseMyDomain( selectedSite.slug, {
+					domain: get( context, 'params.suggestion', '' ),
+				} )
```

Comment:
I wasn't really sure how to test this part, apart from manually changing the code to pass this `if` test

Reply:
You can only test it with a VIP site.